CODEOWNERS

```diff
@@ -62,26 +62,51 @@ Jenkinsfile @cilium/ci
 Jenkinsfile.nightly @cilium/ci
 monitor/ @cilium/monitor
 monitor/payload @cilium/api
+pkg/annotation @cilium/kubernetes
 pkg/apierror/ @cilium/api
 pkg/apipanic/ @cilium/api
 pkg/apisocket/ @cilium/api
 pkg/bpf/ @cilium/bpf
+pkg/byteorde/ @cilium/bpf
+pkg/client @cilium/api
+pkg/comparator @cilium/bpf
```

Comment:
I'll leave it with janitor. Looking at it now, I don't remember why I picked bpf, actually :/

Reply:
I think it's because during startup, if there is a diff between the definition of the map that is being restored from a previous run and the current version of Cilium, then this package is used to pretty-print log messages to show the differences. (Unless this has been changed/fixed recently)

Reply:
That sounds familiar. Probably it :)